### ISSUE TITLE
Bombchus Considered In Logic documentation

### ISSUE THREAD

**Issue by OxleyS** [AmazingAmpharos/OoT-Randomizer#171]
On the current Dev branch (HEAD da47a9e), the following is in the README, under "Bombchus Are Considered In Logic":

> Finding Bombchus will be considered as logical progression, and having found any Bombchus will be the condition to play Bombchu Bowling or buy Bombchu refills from shops.

I think the current documentation does not make clear that Bombchus are considered as bombs for the purpose of logic (e.g. my current seed requiring me to bomb the eyes in Dodongo's Cavern with Bombchus before I've found a bomb bag).

**Comment by AmazingAmpharos** [comment]
Bombchus and Bombs are considered separately with this flag set. Some tasks, such as defeating King Dodongo himself, need specifically Bomb Bag as Bombchus either don't work or require extreme trickery to work. Others, such as hitting the back switch in the Spirit Trial of Ganon's Castle, require specifically Bombchus. Most places both types of explosives can work, and if it's not obvious how to use Bombchus, you can just kinda pull one out and stand in place to let it blow up in Link's hands (can require setting them down right before they explode to do things like open grottos, especially as the taller adult). So that being the case, no, Bombchus are not considered Bombs with this flag, but most (though not all) places you would conventionally use Bombs you could also use Bombchus.

**Comment by OxleyS** [comment]
Thanks for the clarification, my mistake. I think it would be helpful to have a quick bit about this in that section, perhaps reword it to:
> Finding Bombchus will be considered as logical progression, and many (but not all) places where you would normally use Bombs will become in-logic. Also, having found any Bombchus will be the condition to play Bombchu Bowling or buy Bombchu refills from shops.
